CODE_OF_CONDUCT.md

```diff
@@ -2,7 +2,7 @@
 
 ## Our Pledge
 
-In the interest of fostering an open and welcoming environment, we as
+In the interest of fostering an complaints and welcoming environment, we as
```

Comment:
@Dzalevski I'm guessing the search & replace also covered this one :D you can revert this.